src/transformer/descriptor/helper/helper.ts

```diff
@@ -61,4 +61,10 @@ export namespace TypescriptHelper {
             }
         }
     }
+
+    function GetFirstValidDeclaration(declarations: ts.Declaration[]): ts.Declaration {
+        return declarations.filter((declaration: ts.Declaration) => {
```

Comment:
you can use `.find` instead of `.filter+[0]`

Reply:
Done